src/components/InstanceMonitoring/InstanceMonitoringCard.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Icon } from 'carbon-components-react';
+import { InstanceMonitoringContent } from './InstanceMonitoringContent';
+
+export class InstanceMonitoringCard extends Component {
+  render() {
+    const instanceCardProps = {
+      className: 'instance-card',
+    };
+    return (
+      <section {...instanceCardProps}>
```

Comment:
Let’s accept custom CSS class names in the top-level node - We use `classnames` library to achieve this.